### PR TITLE
入力値チェックを修正しました。

### PR DESCRIPTION
**Pull request by uedamiku**

### CODE REVIEW

File: src/main/java/com/example/controller/ShowTopPage.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class ShowTopPage {
 	 * @param orderId 注文の主キー
 	 * @return 注文確認画面
 	 */
-	@GetMapping("/top")
+	@GetMapping("")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShowTopPage.java:17]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 2]
```diff
@@ -45,8 +45,11 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
 
 		User exitUser = registerUserService.searchByEmail(form.getEmail());
 
+		if (("").equals(form.getFirstName()) || ("").equals(form.getLastName())) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 3]
```diff
@@ -45,8 +45,11 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+			result.rejectValue("lastName", null, "名前を入力してください");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 4]
```diff
@@ -45,8 +45,11 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 5]
```diff
@@ -45,8 +45,11 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
 		if (("").equals(form.getEmail())) {
-			result.rejectValue("email", null, "メールアドレスは必須です");
+			result.rejectValue("email", null, "メールアドレスを入力してください");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 6]
```diff
@@ -57,6 +60,9 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
 			result.rejectValue("passwordConfirmation", null, "パスワードと確認用パスワードが不一致です");
 		}
 
+		if (("").equals(form.getPref()) || ("").equals(form.getMunicipalities()) || ("").equals(form.getAddress())) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 125).

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 7]
```diff
@@ -57,6 +60,9 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+			result.rejectValue("pref", null, "住所を入力してください");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 8]
```diff
@@ -57,6 +60,9 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 9]
```diff
@@ -40,6 +40,10 @@ public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model m
 		if (("").equals(form.getDestinationEmail())) {
 			rs.rejectValue("destinationEmail", null, "メールアドレスは必須です");
 		}
+
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 10]
```diff
@@ -40,6 +40,10 @@ public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model m
 		if (("").equals(form.getDestinationEmail())) {
 			rs.rejectValue("destinationEmail", null, "メールアドレスは必須です");
 		}
+		
+		if (("").equals(form.getDestinationPref()) || ("").equals(form.getDestinationMunicipalities()) || ("").equals(form.getDestinationAddress())) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 158).

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 11]
```diff
@@ -40,6 +40,10 @@ public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model m
...
+			rs.rejectValue("destinationPref", null, "住所を入力してください");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 12]
```diff
@@ -40,6 +40,10 @@ public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model m
...
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/common/SecurityConfig.java [hunk 13]
```diff
@@ -40,7 +40,7 @@ WebSecurityCustomizer webSecurityCustomizer() {
 	@Bean
 	SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		http.authorizeHttpRequests(ahr -> ahr
-				.requestMatchers("/login-user/to-login", "/register-user/to-register", "/login-user/login-user",
+				.requestMatchers("/","/login-user/to-login", "/register-user/to-register", "/login-user/login-user",
```
Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 132).

Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/common/SecurityConfig.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**